### ISSUE TITLE
AIO, Google docs exporter, Etherpad Exporter

### ISSUE THREAD

**Issue by Denubis** [carpentries/styles#432]
In preparation for migrating (As per discussion in #431) my etherpad (and now google-docs) exporters to the universal style, what features, capabilities, and endpoints are desired for us to quickly produce note-taking documents for participants in our workshops? Do any of these features not exist in the demo documents:

## Instructor training etherpad:
https://carpentries.github.io/instructor-training/etherpad/index.html -> https://pad.carpentries.org/2019-07-18-ttt-macquarie

## Python google docs
https://denubis.github.io/python-novice-inflammation/ -> http://bit.ly/resbaz-python

## SQL google docs 
I already know I need to fix sql code and table detection. It was somewhat embarrassing to learn I missed that when running the workshop. 
https://denubis.github.io/sql-novice-survey/ -> http://bit.ly/resbaz-sql

Pinging @fmichonneau and @maxim-belkin on this one. Let's get our specification written first, instead of in an endless iterating pull request.

**Comment by Denubis** [comment]
Reading @brownsarahm's https://carpentries.org/blog/2018/09/teaching-tip-exercise-discussion/ -- is it worth exporting all code as code files to support load magic in lessons? (As an aside, do we want to use load magic as the basis for faded examples? With this infrastructure,and https://stackoverflow.com/a/16769695/263449 we can pack various exercises up into a zip file. Obviously this is not the place to discuss [faded examples](https://carpentries.github.io/instructor-training/05-memory/#guided-practice-at-work) in workshops but this may be an opportunity to build infrastructure to use what we teach.

**Comment by lexnederbragt** [comment]
This is cool stuff! Auto-generating slides (e.g. Google Slides or markdown-based) with Questions, Objectives, Key Points and exercises would also be very helpful...

**Comment by beacurious** [comment]
When you did the instructor training that I was part of I noticed that you often (occasionally?) wanted to show an image that was in the notes (e.g. mind map examples, cartoons), but you didn't want to show the notes themselves. I think you had a separate browser tab with them loaded (or a document, I can't remember). Is this something that could be help with this tool?

**Comment by Denubis** [comment]
Yes, that's why the tool exposes urls of images, so they can be opened in new tabs. In the google docs version, they're inline.
